menuflow/nodes/http_request.py

```diff
@@ -113,9 +115,20 @@ async def request(self, session: ClientSession, middleware: HTTPMiddleware) -> T
         request_params_ctx = self._context_params
         request_params_ctx.update({"middleware": middleware})
 
-        response = await session.request(
-            self.method, self._url, **request_body, trace_request_ctx=request_params_ctx
-        )
+        try:
+            timeout = ClientTimeout(total=self.config["menuflow.timeouts.http_request"])
+            response = await session.request(
+                self.method,
+                self._url,
+                **request_body,
+                trace_request_ctx=request_params_ctx,
+                timeout=timeout,
+            )
+        except Exception as e:
+            self.log.exception(f"Error in http_request node: {e}")
+            o_connection = await self.get_case_by_id(id=str(500))
+            await self.room.update_menu(node_id=o_connection, state=None)
+            return 500, e
```

Comment:
Are you sure that error returned not make a other bug? the function definition tell us that return is `Tuple(int, str)`